jdi-light-mobile-tests/src/main/java/io/github/epam/custom/Result.java

```diff
@@ -0,0 +1,7 @@
+package io.github.epam.custom;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Result extends DataClass<Result> {
+    public String name, description, link;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)